static/js/dashboard.js

```diff
@@ -317,6 +318,88 @@ function switchFrom(el1, el2, el3, el4) {
 
 var firstSwitch = true
 
+function initValidatorCountdown(validatorIndex, queueId, ts) {
+  var now = Math.round(new Date().getTime() / 1000)
+  var secondsLeft = ts - now
+  var seconds = secondsLeft % 60
+  var min = ((secondsLeft - seconds) / 60) % 60
+  var hour = Math.round((secondsLeft - seconds - min * 60) / 3600) % 24
+  var days = Math.round((secondsLeft - seconds - min * 60 - hour * 60 * 60) / 86400)
+  setValidatorCountdown(validatorIndex, queueId, days, hour, min, seconds)
+
+  if (!countdownIntervals.has(validatorIndex)) {
+    countdownIntervals.set(
+      validatorIndex,
+      setInterval(function () {
+        if (hour <= 0 && min <= 0 && seconds <= 0 && days <= 0) {
+          clearInterval(countdownIntervals.get(validatorIndex))
+          return
+        }
+
+        if (seconds === 0 && min === 0 && hour === 0 && days > 0) {
+          hour = 24
+          days -= 1
+        }
+
+        if (seconds === 0 && min === 0 && hour > 0) {
+          min = 60
+          hour -= 1
+        }
+
+        if (seconds === 0 && min > 0) {
+          seconds = 60
+          min -= 1
+        }
+
+        seconds -= 1
+
+        setValidatorCountdown(validatorIndex, queueId, days, hour, min, seconds)
```

Comment:
Instead of passing days/hour/... why not simply reduce the secondsLeft by the interval time and pass the secondsLeft. This way you could remove all the complecated -= logic above. 

And only format the secondsLeft (via the luxon duration object) when you need it for creating the tooltip.